Puppetfile

```diff
@@ -79,3 +79,8 @@ mod 'datadog/datadog_agent', '1.6.0'
 
 # Used for grabbing certificates for jenkins.io
 mod 'danzilio/letsencrypt', '1.0.0'
+
+# For managing ldap
+mod 'camptocamp/openldap', '1.14.0'
```

Comment:
This might need to go into `.fixtures.yml` too so rspec-puppet knows how to handle it with tests